xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/client/server/ServerInfoResultTests.java

```diff
@@ -335,7 +335,7 @@ private ServerInfo serverInfo(final String completeLedgers) {
         )
         .threads(UnsignedLong.valueOf(6))
         .build())
-      .loadFactor(UnsignedInteger.ONE)
+      .loadFactor(1.0)
```

Comment:
A little more testing here could be useful. I'd suggest a few tests with differing precisions in JSON, as well as a test without any decimal precision, just to make sure this change covers everything rippled will send back

Reply:
Hey @tommq, just wanted to make sure you saw this comment.